ui/app/components/clients/table.hbs

```diff
@@ -0,0 +1,59 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{#if @data}}
```

Comment:
Should this be updated to `paginatedTableData`? Looking at the different empty state messages, I'm wondering if relying on just `this.args.data` isn't sufficient?